docs/scenario-dashboard.md

```diff
@@ -0,0 +1,228 @@
+    """main uut driver function"""
+    # ...
+    simulator = get_simulator()
+    # ...
+```
+
+#### Results  
+Finally, at the end of main script, you will create a `Result` object and store it with the
+`set_result` function. These metrics are defined by the user as desired. They will be surfaced
+to the Batch Scenario Manager results page, but are not required for the batch processing to run properly.
+
+```python
+from monodrive.jobs import set_result, Result, ResultMetric
+
+def main():
+    """main uut driver function"""
+
+    # ...
+
+    result = Result()
+    result.pass_result = True
+    result.metrics.append(
+        ResultMetric(
+            name='max_lane_deviation',
+            score=0.123
+        )
+    )
+    set_result(result)
+```
+ 
+#### Example
+For a full working UUT, please check out this [collision avoidance](https://github.com/monoDriveIO/monodrive-python-client/tree/master/examples/collision_avoidance)
+use case, written in Python. This directory includes both a replay and closed loop mode example.
+
+*(C++ example coming soon)*
+
+
+## Configurable Selections
+
+### Batch Scenario Manager Menu
+
+The Batch Scenario Manager Menu shows a summary of all the configurations that have been selected in the configuration selection process: vehicles and sensor configurations, scenario configurations, and weather configurations. On the bottom right corner of the menu, an user may use "Select Configurations" to make and/or edit selections.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_starting_menu.png"/>
+  </div>
+
+
+### Vehicles & Sensors
+
+When using closed loop simulation mode, the ego vehicle will be defined in the scenario file with the tag, `ego`. When making vehicle selections, select or add the vehicle(s) that will be used in the closed loop scenario file. The vehicle's `class_path` and `body_color` will be automatically updated in the vehicle object tagged `ego`. For example, `scenario_aebs.json` file in the UUT Python Example under [Configuring UUT](/scenario-dashboard/#connecting-your-code).
+
+```json
+{
+  "trigger_boxes": [],
+  "vehicles": [
+    {
+      "body_color": "Carpaint_DarkGrey",
+      "class_path": "/Game/Vehicles/sedan_monoDrive_02.sedan_monoDrive_02_C",
+      "tags": [
+        "vehicle",
+        "dynamic",
+        "car",
+        "ego"
+      ],
+      ...
+```
+
+When using replay simulation mode, users can select multiple vehicle and sensor configurations to run. Select a vehicle configuration from monoDrive's vehicle models or add a custom vehicle as the ego vehicle in the simulation. The custom vehicle requires the directory path to the vehicle asset and enables users to change the color of the vehicle. By default all monoDrive's vehicle models spawn with white paint; if an user would like to customize the color on a monoDrive vehicle asset, select "Add New Vehicle" and use the path to the monoDrive vehicle asset and change it's color.
```

Comment:
Is this out of date? I think this can be summarized with something along the lines of "you are setting the vehicle marked with the ego tag"

Reply:
Do we not want to include how to change vehicle color in the documentation, or is it an unnecessary tip?